pkgs/by-name/gl/glitchtip/package.nix

```diff
@@ -13,6 +15,24 @@ let
     packageOverrides = final: prev: {
       django = final.django_5;
       django-extensions = prev.django-extensions.overridePythonAttrs { doCheck = false; };
```

Comment:
Side quest: I removed this in paperless-ngx recently. Maybe it is no longer required?

Reply:
Nice, seems to work, thanks!